cwltool/load_tool.py

```diff
@@ -48,7 +48,7 @@ def fetch_document(argsworkflow,  # type: Union[Text, Dict[Text, Any]]
     workflowobj = None  # type: CommentedMap
     if isinstance(argsworkflow, string_types):
         split = urllib.parse.urlsplit(argsworkflow)
-        if split.scheme:
+        if split.scheme and split.scheme in [u'http',u'https',u'file']:
```

Comment:
Cool, can you link to that discussion in the code and add what you said as a comment?